docs/source/release/v6.2.0/reflectometry.rst

```diff
@@ -5,18 +5,13 @@ Reflectometry Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
 Reflectometry
 -------------
 
 Improvements
 ############
 - :ref:`LRSubtractAverageBackground <algm-LRSubtractAverageBackground-v1>` now optionally uses weighted error for background with new input property ErrorWeighting.
 - :ref:`LiquidsReflectometryReduction <algm-LiquidsReflectometryReduction-v1>` now optionally uses weighted error for background with new input property ErrorWeighting with the default as False.
-
 - Version 2 of :ref:`ConvertToReflectometryQ <algm-ConvertToReflectometryQ>` has been added which assumes detectors are at :math:`2\theta` rather than :math:`\theta_f`. Use version 1 if you require detectors at :math:`\theta_f`.
```

Comment:
We're missing a release note here:
https://github.com/mantidproject/mantid/commit/fba74436d7592d60aa357468b5fd7aaf5db59933#diff-d3e43f83d066d0cc621c971c101d990224e9a9e387b6cd47e8eea834192129aa
```
- The detectors positions in the INTER IDF were changed from theta to two theta.
```

Reply:
Other than that I'm happy with this